src/handle_action.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#ifndef CATA_SRC_HANDLE_ACTION_H
+#define CATA_SRC_HANDLE_ACTION_H
+
+void handbrake();

```

Comment:
Oops, didn't know that `handle_action.h` didn't exist. Based on what `handbrake` does it's probably better to move this to `vehicle.h` and move the definition of `handbrake` to `vehicle_use.cpp`?

Reply:
;-) Sure. Thanks for taking the time to help me with that. I'll see what I can do.